app/experimenter/experiments/constants/nimbus.py

```diff
@@ -206,6 +206,29 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_NOT_TCP_STUDY = NimbusTargetingConfig(
+    name="Exclude users in the TCP revenue study",
+    slug="not_tcp_study",
+    description="Exclude users with certain search codes set",
+    targeting="!'browser.search.param.google_channel_us'|preferenceValue('')|regExpMatch"
+    "('^[ntc]us5$') && !'browser.search.param.google_channel_row'|preferenceValue('')|"
+    "regExpMatch('^[ntc]row5$')",
```

Comment:
Just copying over the targeting from https://github.com/mozilla/messaging-system-inflight-assets/pull/235/files#diff-c41aca0e731ed15191b531d6cfc8434c4b70695a96ad97f86b3c96946bbcfef9R355-R356